SimpleWAWS/Code/CsmExtensions/CsmSubscriptionExtensions.cs

```diff
@@ -22,18 +46,19 @@ public static partial class CsmManager
                 if (deleteBadResourceGroups)
                 {
                     var deleteBadResourceGroupsTasks = csmResourceGroups.value
-                        .Where(r => r.tags != null
-                                    && ((r.tags.ContainsKey("Bad") || 
-                                    (subscription.Type==SubscriptionType.AppService?!r.tags.ContainsKey("FunctionsContainerDeployed"): !r.tags.ContainsKey(Constants.SubscriptionType)))
-                                    && (!r.tags.ContainsKey("UserId"))
+                        //Some orphaned resourcegroups can have no tags. Okay to clean once in a while since they dont have any sites either
+                        .Where(r => ((r.tags == null && IsSimpleWawsResourceName(r)) 
```

Comment:
The changes below are mostly formatting , I thought would make the logic easier to read. I'll revert them.
The main changes are adding  `((r.tags == null && IsSimpleWawsResourceName(r))` on top 
and 
`  && (!r.tags.ContainsKey("UserId")) `
down below